Dockerfile

```diff
@@ -1,5 +1,5 @@
 ARG UBI_IMAGE=registry.access.redhat.com/ubi7/ubi-minimal:latest
-ARG GO_IMAGE=briandowns/rancher-build-base:v0.1.1
+ARG GO_IMAGE=golang:1.14.7
```

Comment:
@rmweir do we want to use upstream golang here and not a rancher image ?

Reply:
I would prefer also to use internal container image. We should not trust upstream images. @rmweir Please let me know about possible options to use

Reply:
We can probably use rancher/hardened-build-base  which I think is the replacement for `rancher-build-base`. However,I'm not sure using the official golang image is any less secure than our own images. This image is just used as a build environment for our own image.

Reply:
I replaced image to `rancher/hardened-build-base:v1.14.2-amd64`